src/FsAutoComplete/Parser.fs

```diff
@@ -170,6 +176,32 @@ module Parser =
 
       next.Invoke(ctx))
 
+  let configureOTel =
+    Invocation.InvocationMiddleware(fun ctx next ->
+      let serviceName = FsAutoComplete.Utils.Tracing.serviceName
+      let fcsServiceName = "fsc"
+      let fcsProfileName = "fsc_with_env_stats"
+      let version = FsAutoComplete.Utils.Version.info().Version
+
+      tracerProvider <-
+        Sdk
+          .CreateTracerProviderBuilder()
+          .AddSource(serviceName, fcsServiceName, fcsProfileName)
+          .SetResourceBuilder(
+            ResourceBuilder
+              .CreateDefault()
+              .AddService(serviceName = serviceName, serviceVersion = version)
+          )
+          .AddZipkinExporter()
```

Comment:
This can be parameterized like so: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/AspNetCore/Program.cs#L64-L101

Reply:
So we have to have all three exporters? Or is there a way to use just `AddOtlpExporter` and send to Jaeger/Zipkin?

Reply:
the oltp exporter this pushes to could be configured to push to jaeger/zipkin (see [configuration](https://opentelemetry.io/docs/collector/configuration/#exporters)), but that would mean 2 components instead of 1. I do think that's the most agnostic way forward, tho.

Reply:
Jaeger accepts OTLP natively, so it's really just Zipkin that'd need a zipkin exporter dependency. Although in practice so few people use zipkin that it's probably worth dropping

Reply:
Yeah took me a while to figure out the jaeger configuration.  [So many freaking ports to open](https://opentelemetry.io/docs/instrumentation/net/exporters/)!

Reply:
OK pushed with the generic exporter. Also left some docs in the readme on how to use it.